### PR TITLE
Rewrite chat logs handling to use log4j2

### PR DESCRIPTION
**Pull request by idoash4**
To use the custom log4j config simply pass it as an option when launching the server jar(-Dlog4j.configurationFile=log4j2.xml). With the default config provided by Mojang chat messages will still show up in console and will be logged to the default logs. The custom config will create a new folder logs/ChatLogs and will save all chat logs there, separated from the server logs.

### CODE REVIEW

**Comment by ProgrammerDan** [comment]
Looks pretty good.

I believe we already use a custom log4j2.xml, so we could look to blend these.

Level of testing accomplished, @BlackXnt ?

**Comment by idoash4** [comment]
Fully tested. Not much that can go wrong.

**Comment by erocs** [comment]
We haven't used a custom log4j.xml for 3.0 like we did in 2.0. I brought it up but nothing has been changed yet.

**Comment by ProgrammerDan** [comment]
In that case this one looks fairly safe; any thoughts on it @erocs?

**Comment by idoash4** [comment]
bump

File: pom.xml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>vg.civcraft.mc.civchat2</groupId>
 	<artifactId>CivChat2</artifactId>
-	<packaging>jar</packaging>
-	<version>1.29</version>
+	<packaging>jar</packaging>1
```
Comment (erocs) [inline pom.xml]: That 1 doesn't mess up the pom?

Comment (idoash4) [reply pom.xml]: It does. Don't ask me how it got there...

**Pull request merged**